c10/hammerblade/CMakeLists.txt

```diff
@@ -120,8 +120,11 @@ endforeach()
 install(FILES ${CMAKE_BINARY_DIR}/c10/hammerblade/impl/hammerblade_cmake_macros.h
   DESTINATION include/c10/hammerblade/impl)
 
+file(GLOB KERNEL_FILES ${CMAKE_CURRENT_BINARY_DIR}/riscv/kernel.riscv
+                       ${CMAKE_CURRENT_BINARY_DIR}/riscv/*.dis)
+
 if(NOT USE_HB_EMUL)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/kernel.riscv DESTINATION riscv)
+  install(FILES ${KERNEL_FILES} DESTINATION riscv)
```

Comment:
@vb000  I think after this change, `torch/riscv` is missing when building from a clean clone. somehow these files are not "installed" correctly

Reply:
@zzy82518996 you saw similar issues I believe?

Reply:
Yes, I also came across the same issue.

Reply:
Not sure why this is the case with fresh builds. I'll look into it...

Reply:
not too sure as well, but it seems to me this `install` command didnt do anything. I was wondering if the setting the entire path `riscv/` as `BYPRODUCTS` is not supported by cmake?

Btw, I dont have `ninja` installed, is that a must have for this?

Reply:
No, I think. The byproducts thing only builds that too only `clean` target because we rebuild every single time because of the cross compilation issues. Could you confirm if they correctly built in `build/c10/hammerblade/riscv/`?

Reply:
Yes, everything is there in `build/c10/hammerblade/riscv/`, including all the` *.dis`
but none of them are copied over to `torch/riscv/`

Reply:
You know what to do... just copy then to torch/riscv ;). JK, I'll try to figure out the issue.

I tried removing my CMakeCache, but still it seems to install fine! So, I'll try a fresh build...

Reply:
yeah we are manual copying them as a work around of course! ;)

can you share your cmake version maybe? we are using `cmake version 3.13.3`

Reply:
`cmake3 version 3.17.1`

Reply:
maybe I need a version bump

which version are you using? @zzy82518996

Reply:
my cmake version is 3.18.0
my cmake3 version is 3.17.3

Reply:
this is inside my python virtual environment.